app/models/nic/managed.rb

```diff
@@ -128,7 +128,7 @@ def failure(msg, backtrace = nil, dest = :base)
 
     # we must also clone host object so we can detect host attributes changes
     def setup_clone
-      return if new_record?
+      return if new_record? or host.new_record?
```

Comment:
The test causing it does not make sense. The interface should not be saved (it's inconsistent state that we are trying to handle if we use create). I'll try to trace it further to find out whether we could fix it even for such weird situation but fixing the test is OK for this PR.